### PR TITLE
Implement collector

### PR DESCRIPTION
**Pull request by Hosshii**
**What this PR does / why we need it**:
Implement insight collector.

**Which issue(s) this PR fixes**:

ref #1142

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/ops/insightcollector/deployments.go [hunk 1]
```diff
@@ -49,311 +45,86 @@ package insightcollector
...
+				},
+			},
+		})
+		if err != nil {
+			return nil, err
+		}
+		if len(d) == 0 {
+			// get all deployments in range
+			break
+		}
```
Comment (knanao) [inline pkg/app/ops/insightcollector/deployments.go:102]: I think it better checking whether being got all deployments or not with cursor.

Comment (Hosshii) [reply pkg/app/ops/insightcollector/deployments.go:102]: I certainly agree . Thank you!

**Comment by nghialv** [comment]
Nice.

/lgtm

**Comment by knanao** [comment]
Go bold!
/approve

**Pull request merged**